app/src/main/java/com/github/se/eventradar/model/event/Event.kt

```diff
@@ -56,17 +56,17 @@ data class Event(
     map["ticket_price"] = ticket.price
     map["ticket_quantity"] = ticket.capacity
     map["main_organiser"] = mainOrganiser
-    map["organisers_list"] = organiserList.toList()
-    map["attendees_list"] = attendeeList.toList()
+    map["organisers_list"] = organiserSet.toList()
+    map["attendees_list"] = attendeeSet.toList()
     map["category"] = category.name
     return map
   }
 }
 
-private fun getSetOfStrings(data: Any?): Set<String> {
+private fun convertToMutableSetOfStrings(data: Any?): MutableSet<String> {
```

Comment:
nit: This seems to be reused twice. Would be nice to put this in a reusable place, but not a priority.